prototype/pom.xml

```diff
@@ -296,11 +300,31 @@
         <artifactId>org.eclipse.emf.ecore.xmi</artifactId>
         <version>${emf.ecore.xmi.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.eclipse.emf</groupId>
+        <artifactId>org.eclipse.emf.ecore.change</artifactId>
+        <version>${emf.compare.change.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.emf</groupId>
+        <artifactId>org.eclipse.emf.compare</artifactId>
+        <version>${emf.compare.version}</version>
+      </dependency>
+      <dependency>
```

Comment:
Is there any way that we can avoid guava? It's a versioning nightmare as they always bump the major version.

Reply:
Not for now, as this is a dependency of emf compare. As they build with tycho, their POMs don't have a dependency section. Thus we need to add their transient dependencies. 

When I find the time, I want to create PR for them to replace Guava, as this seems to be a historic dependency from the times before Function, Supplier, Lists and so on. So, it can be completely replaced with standard Java means.